test/EFCore.PG.FunctionalTests/Query/MathQueryTest.cs

```diff
@@ -34,155 +32,133 @@ public MathQueryTest(MathQueryNpgsqlFixture fixture, ITestOutputHelper testOutpu
+                    .ToArray();
+
+            AssertContainsSql("SELECT GREATEST(x.\"Int\", x.\"Int\")");
         }
 
         [Fact]
         public void Max_long_long()
         {
-            using (var ctx = Fixture.CreateContext())
-            {
-                var _ =
-                    ctx.MathTestEntities
-                       .Select(x => Math.Max(x.Long, x.Long))
-                       .ToArray();
-
-                AssertContainsSql("SELECT GREATEST(x.\"Long\", x.\"Long\")");
-            }
+            using var ctx = Fixture.CreateContext();
+            var _ =
+                ctx.MathTestEntities
+                    .Select(x => Math.Max(x.Long, x.Long))
+                    .ToArray();
+
+            AssertContainsSql("SELECT GREATEST(x.\"Long\", x.\"Long\")");
         }
 
         [Fact]
         public void Max_float_float()
         {
-            using (var ctx = Fixture.CreateContext())
-            {
-                var _ =
-                    ctx.MathTestEntities
-                       .Select(x => Math.Max(x.Float, x.Float))
-                       .ToArray();
-
-                AssertContainsSql("SELECT GREATEST(x.\"Float\", x.\"Float\")");
-            }
+            using var ctx = Fixture.CreateContext();
+            var _ =
+                ctx.MathTestEntities
+                    .Select(x => Math.Max(x.Float, x.Float))
+                    .ToArray();
+
+            AssertContainsSql("SELECT GREATEST(x.\"Float\", x.\"Float\")");
         }
 
         [Fact]
         public void Max_double_double()
         {
-            using (var ctx = Fixture.CreateContext())
-            {
-                var _ =
-                    ctx.MathTestEntities
-                       .Select(x => Math.Max(x.Double, x.Double))
-                       .ToArray();
-
-                AssertContainsSql("SELECT GREATEST(x.\"Double\", x.\"Double\")");
-            }
+            using var ctx = Fixture.CreateContext();
+            var _ =
+                ctx.MathTestEntities
+                    .Select(x => Math.Max(x.Double, x.Double))
+                    .ToArray();
+
+            AssertContainsSql("SELECT GREATEST(x.\"Double\", x.\"Double\")");
         }
 
         [Fact]
         public void Max_decimal_decimal()
         {
-            using (var ctx = Fixture.CreateContext())
-            {
-                var _ =
-                    ctx.MathTestEntities
-                       .Select(x => Math.Max(x.Decimal, x.Decimal))
-                       .ToArray();
-
-                AssertContainsSql("SELECT GREATEST(x.\"Decimal\", x.\"Decimal\")");
-            }
+            using var ctx = Fixture.CreateContext();
+            var _ =
+                ctx.MathTestEntities
+                    .Select(x => Math.Max(x.Decimal, x.Decimal))
+                    .ToArray();
+
+            AssertContainsSql("SELECT GREATEST(x.\"Decimal\", x.\"Decimal\")");
         }
 
         //[Fact]
```

Comment:
Why disabled?

Reply:
No idea... maybe open an issue for us to look into it?

Reply:
Let's do it.